packages/analytics-js/src/services/StoreManager/storages/storageEngine.ts

```diff
@@ -57,23 +59,33 @@ const configureInMemoryStorageEngine = (options: Partial<IInMemoryStorageOptions
   defaultInMemoryStorage.configure(options);
 };
 
+/**
+ * Configure in memory storage singleton
```

Comment:
comment is misleading should be session storage instead

Reply:
Addressed